src/item/itemRepository.ts

```diff
@@ -19,79 +19,77 @@ export class ItemRepository implements IItemRepository {
   }
 
   async createCategory(categoryName: string) {
-    const repo = this.dataSource.getRepository(Category);
-    await repo.insert({
+    await CategoryModel.create({
       name: categoryName
     })
   }
 
   async deleteItem(itemId: number) {
-    const repo = this.dataSource.getRepository(Item);
-    await repo.delete({id: itemId})
+    await ItemModel.destroy({where: {id: itemId}})
   }
 
   async getItemsList(limit: number, offset: number) {
-    const list = await this.dataSource.getRepository(Item)
-    .createQueryBuilder("i")
-    .select([
-      "i.id",
-      "i.name",
-      "i.count",
-      "i.price",
-      "i.photo",
-      "c.id AS category_id",
-      "c.name AS category_name",
-    ])
-    .innerJoin("i.category", "c")
-    .limit(limit)
-    .offset(offset)
-    .getRawMany(); 
-  return list;
+    const list = await ItemModel.findAll({
+      attributes: [
+        "id",
+        "name",
+        "count",
+        "price",
+        "photo",
+        [Sequelize.col("category.id"), "category_id"],
+        [Sequelize.col("category.name"), "category_name"],
+      ],
+      include: [
+        {
+          model: CategoryModel,
+          as: "category",
+          attributes: [],
+        },
+      ],
+      limit,
+      offset,
+      raw: true,
+    });
+  
+    return list;
   }
 
   async changeCount(count: number, itemId: number) {
-    const repo = this.dataSource.getRepository(Item);
-    await repo.update({id: itemId}, {count: count})
+    await ItemModel.update({count: count}, {where: {id: itemId}})
   }
 
   async changePrice(price: number, itemId: number) {
-    const repo = this.dataSource.getRepository(Item);
-    await repo.update({id: itemId}, {price: price})
+    await ItemModel.update({price: price}, {where: {id: itemId}})
   }
 
   async findItem(itemId: number) {
-    const repo = this.dataSource.getRepository(Item);
-    return repo.findOne({where: {id: itemId}})
+    return ItemModel.findOne({where: {id: itemId}})
   }
 
   async priceFilter(sortBy: "ASC" | "DESC") {
-    const repo = this.dataSource.getRepository(Item);
-    const items = await repo
-    .createQueryBuilder("i")
-    .select("*")
-    .orderBy("i.price", sortBy)
-    .getRawMany();
+    const items = await ItemModel.findAll({
+      attributes: ["*"],
+      order: [["price", sortBy]],
+      raw: true,
+    });
   return items;
   }
 
   async dateFilter(sortBy: "ASC" | "DESC") {
-    const repo = this.dataSource.getRepository(Item);
-    const items = await repo
-    .createQueryBuilder("i")
-    .select("*")
-    .orderBy("i.created_at", sortBy)
-    .getRawMany();
+    const items = await ItemModel.findAll({
+      attributes: ["*"],
```

Comment:
if u want to select all the column, u can just delete attributes fields. Sequelize automatically will select all the columns